webpack.config.js

```diff
@@ -8,27 +8,26 @@ require('dotenv').config({silent: true});
 var outputPath = 'public/builds';
 
 function buildPluginList() {
-  var plugins = [
-    //new CleanWebpackPlugin(outputPath),
-    //new ExtractTextPlugin('style.css')
-  ];
+  var plugins = [];
 
   if(process.env.NODE_ENV === 'production') {
     console.log('building production plugin list');
-    plugins = plugins.concat([
-      new webpack.DefinePlugin({
-        API_BASE_URL: JSON.stringify('')
-      }),
+    plugins.concat([
```

Comment:
concat() returns a new array. It doesn't modify the array in place like push() does.